test/integration/lib/render.js

```diff
@@ -125,26 +140,27 @@ exports.run = function (implementation, ignores, render) {
 
             png.data = data;
 
-            if (process.env.UPDATE) {
+            let writeExpected = !!process.env.UPDATE;
+            try {
+                stats = fs.statSync(expected, fs.R_OK | fs.W_OK);
+                if (!stats.isFile()) throw new Error();
+
+            } catch (e) { // no expected.png, create it
+                writeExpected = true;
```

Comment:
Can we add a message to the console when we're updating this to at least give the user some indication of what is happening